### DISCUSSION TITLE
How and What do I do with the Max flow rate test?

### DISCUSSION THREAD

**Discussion by jkjerde** [OrcaSlicer/OrcaSlicer#486] [General]
excuse my ignorance but I have never done this before.  I Printed off the test but don't know what to do with it. 

Thanks

**Comment by Manfredo1952** [comment]
I guessa a good starting point would be to read here https://www.printables.com/model/405354-volumetric-flow-test-200-660mms

**Comment by survivingscout** [comment]
ModBot on youtube also has a short (~7 minute) how-to video on the VFA test: ["Find Your 3d Printers MAX Output"](https://youtu.be/Hr5ZR9lVCRA)

**Comment by chuckkay** [comment]
what about the other test called VFA (right after Max Flow Test) that prints an tall star shape, how do you use and read the results on that one?

**Reply by chuckkay** [reply]
hello

**Reply by rickyspanish8** [reply]
interested on that answer

**Reply by tokamac** [reply]
The VFA test seems to have been disabled lately with no explanation. Will we ever know?
